api_client/python/timesketch_api_client/client.py

```diff
@@ -66,8 +73,9 @@ class TimesketchApi:
     DEFAULT_OAUTH_LOCALHOST_URL = "http://localhost"
     DEFAULT_OAUTH_API_CALLBACK = "/login/api_callback/"
 
-    # Default retry count for operations that attempt a retry.
-    DEFAULT_RETRY_COUNT = 5
+    # Default total attempts (initial request + retries) for operations.
+    # A value of 4 means 1 initial attempt + 4 retries = 5 total attempts.
+    DEFAULT_RETRY_COUNT = 4
 
     def __init__(
```

Comment:
Should we make the retry value configurable via the init? So default to 4 retries but if someone likes 0 or 100 they can choose themselves depending on their needs and deployment.